src/responder/kcm/kcmsrv_ccache_secdb.c

```diff
@@ -818,10 +822,181 @@ static errno_t ccdb_secdb_get_default_recv(struct tevent_req *req,
     return EOK;
 }
 
+static errno_t ccdb_secdb_get_cc_for_uuid(TALLOC_CTX *mem_ctx,
+                                          size_t uuid_list_count,
+                                          const char **uuid_list,
+                                          const char **uid_list,
+                                          struct ccdb_secdb *secdb,
+                                          struct kcm_ccache ***_cc_list)
+{
+    TALLOC_CTX *tmp_ctx;
+    errno_t ret;
+    uid_t uid;
+    char **list;
+    uuid_t uuid;
+    char *uuid_str;
+    char *secdb_key;
+    struct cli_creds cli_cred;
+    struct kcm_ccache **cc_list;
+    int real_count = 0;
+
+    tmp_ctx = talloc_new(NULL);
+    if (tmp_ctx == NULL) {
+        ret = ENOMEM;
+        return ret;
+    }
+
+    cc_list = talloc_zero_array(tmp_ctx, struct kcm_ccache *, uuid_list_count + 1);
+    if (cc_list == NULL) {
+        ret = ENOMEM;
+        goto done;
+    }
+
+
```

Comment:
two empty lines instead of one

Reply:
Fixed.

Reply:
Do we actually have anything in the coding style on this matter?
I'm asking because I also like newlines :)

Reply:
I don't think there is specific statement that we can't use more then two lines. Why would you like to use more than one?